keps/sig-node/3759-sidecar-containers/kep.yaml

```diff
@@ -0,0 +1,53 @@
+title: KEP Template
+kep-number: 3759
+authors:
+  - "@SergeyKanzhelev"
+  - "@matthyx"
+  - "@jpbetz"
+owning-sig: sig-node
+participating-sigs:
+  - sig-scheduler
+  - sig-apps
+status: provisional
+creation-date: 2023-01-19
+reviewers:
+  - "@mrunalp"      # overall
+  - "@ffromani"     # resource management
+  - "@swatisehgal"  # resource management
+  - "@ahg-g"        # SIG Scheduling
+approvers:
+  - "@derekwaynecarr"
+  - "@thockin"
+
+see-also:
+
+replaces:
+  - "keps/sig-node/753-sidecar-containers"
```

Comment:
How come we're replacing that one? A comment could help.

Reply:
added